### PR TITLE
feat(FeedbackDialog.web): add timeout

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: react/features/feedback/components/FeedbackDialog.web.js [hunk 1]
```diff
@@ -176,6 +187,12 @@ class FeedbackDialog extends Component<Props, State> {
         if (typeof APP !== 'undefined') {
             APP.API.notifyFeedbackPromptDisplayed();
         }
+        if (this.props.timeout && this.props.timeout > 0) {
+            this._timeout = setTimeout(() => {
+                this._onCancel();
+                this.props.dispatch(hideDialog());
+            }, this.props.timeout);
+        }
```
Comment (virtuacoplenny) [inline react/features/feedback/components/FeedbackDialog.web.js]: Not sure we want to timeout in 60 if someone is actively typing, for example. Should this be an idle timeout?

Comment (paweldomas) [reply react/features/feedback/components/FeedbackDialog.web.js]: yeah, good point

Comment (paweldomas) [reply react/features/feedback/components/FeedbackDialog.web.js]: that also means it's not a good fix for spot

Comment (virtuacoplenny) [reply react/features/feedback/components/FeedbackDialog.web.js]: or we can fix it on the spot side.

Comment (paweldomas) [reply react/features/feedback/components/FeedbackDialog.web.js]: yes, I guess if we want to track the input it needs to be fixed on the spot side

**Pull request closed without merging**